backend/connector-integration/src/connectors/fiservemea/transformers.rs

```diff
@@ -0,0 +1,906 @@
+    pub message: Option<String>,
+    #[serde(rename = "details")]
+    pub details: Option<Vec<ErrorDetail>>,
+    #[serde(rename = "apiTraceId")]
+    pub api_trace_id: Option<String>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ErrorDetail {
+    pub field: Option<String>,
+    pub message: Option<String>,
+}
+
+impl Default for FiservemeaErrorResponse {
+    fn default() -> Self {
+        Self {
+            code: Some("UNKNOWN_ERROR".to_string()),
+            message: Some("Unknown error occurred".to_string()),
+            details: None,
+            api_trace_id: None,
+        }
+    }
+}
+
+#[derive(Debug, Serialize)]
+pub struct FiservemeaPaymentsRequest<T: PaymentMethodDataTypes> {
+    #[serde(rename = "requestType")]
+    pub request_type: String,
+    #[serde(rename = "merchantTransactionId")]
+    pub merchant_transaction_id: String,
+    #[serde(rename = "transactionAmount")]
+    pub transaction_amount: TransactionAmount,
+    #[serde(rename = "order")]
+    pub order: OrderDetails,
+    #[serde(rename = "paymentMethod")]
+    pub payment_method: PaymentMethod<T>,
+}
+
+#[derive(Debug, Serialize)]
+pub struct PaymentCardSaleTransaction<T: PaymentMethodDataTypes> {
+    #[serde(rename = "requestType")]
+    pub request_type: String,
+    #[serde(rename = "transactionAmount")]
+    pub transaction_amount: TransactionAmount,
+    #[serde(rename = "paymentMethod")]
+    pub payment_method: PaymentMethod<T>,
+    #[serde(rename = "transactionType")]
+    pub transaction_type: String,
+}
+
+#[derive(Debug, Serialize)]
+pub struct PaymentCardPreAuthTransaction<T: PaymentMethodDataTypes> {
+    #[serde(rename = "requestType")]
+    pub request_type: String,
+    #[serde(rename = "transactionAmount")]
+    pub transaction_amount: TransactionAmount,
+    #[serde(rename = "paymentMethod")]
+    pub payment_method: PaymentMethod<T>,
+    #[serde(rename = "transactionType")]
+    pub transaction_type: String,
+}
+
+#[derive(Debug, Serialize)]
+pub struct TransactionAmount {
+    pub total: StringMajorUnit,
+    pub currency: common_enums::Currency,
+}
+
+#[derive(Debug, Serialize)]
+pub struct OrderDetails {
+    #[serde(rename = "orderId")]
+    pub order_id: String,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(tag = "paymentCard")]
+pub struct PaymentMethod<T: PaymentMethodDataTypes> {
+    #[serde(rename = "paymentCard")]
+    pub payment_card: PaymentCard<T>,
+}
+
+#[derive(Debug, Serialize)]
+pub struct PaymentCard<T: PaymentMethodDataTypes> {
+    pub number: RawCardNumber<T>,
+    #[serde(rename = "expiryDate")]
+    pub expiry_date: ExpiryDate,
+    #[serde(rename = "securityCode")]
+    pub security_code: Option<Secret<String>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub holder: Option<String>,
+    #[serde(skip)]
+    _phantom: std::marker::PhantomData<T>,
+}
+
+#[derive(Debug, Serialize)]
+pub struct ExpiryDate {
+    pub month: String,
+    pub year: String,
+}
```

Comment:
I think this should be
```suggestion
pub struct ExpiryDate {
    pub month: Secret<String>,
    pub year: Secret<String>,
}
```

Please look for other sensitive fields that needs to be masked.